src/explorer/Components/MinMaxRefiner.cs

```diff
@@ -15,16 +16,38 @@ public class MinMaxRefiner : ExplorerComponent<MinMaxRefiner.Result>, PublisherC
         private const int MaxIterations = 10;
         private readonly DConnection conn;
         private readonly ExplorerContext ctx;
+        private readonly ResultProvider<MinMaxFromHistogramComponent.Result> histogramMinMaxProvider;
 
-        public MinMaxRefiner(DConnection conn, ExplorerContext ctx)
+        public MinMaxRefiner(
+            DConnection conn,
+            ExplorerContext ctx,
+            ResultProvider<MinMaxFromHistogramComponent.Result> histogramMinMaxProvider)
         {
             this.ctx = ctx;
+            this.histogramMinMaxProvider = histogramMinMaxProvider;
             this.conn = conn;
         }
 
+        public async IAsyncEnumerable<ExploreMetric> YieldMetrics()
+        {
+            var bounds = await ResultAsync;
+
+            if (!(bounds is null))
+            {
+                yield return new UntypedMetric("min", bounds.Min, priority: 10);
```

Comment:
I think that it will be good to define constants for the different priorities of the same metric in a common place.

Reply:
Done 
317ac7f